server/graphql/v2/object/PaymentMethod.js

```diff
@@ -26,6 +29,66 @@ export const PaymentMethod = new GraphQLObjectType({
           return paymentMethod.name;
         },
       },
+      service: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.service;
+        },
+      },
+      type: {
+        type: GraphQLString,
+        resolve(paymentMethod) {
+          return paymentMethod.type;
+        },
+      },
```

Comment:
I would like to change that. We discussed about these `type` and `service` being not great to expose. I will propose something later.